flask/app/utils.py

```diff
@@ -306,3 +307,46 @@ def find(collection: Iterable[Mapping[str, Any]], pred: Callable):
         return next((item for item in collection if pred(item)))
     except StopIteration:
         return None
+
+
+def get_current_user():
+    """if the user is an admin or login is disabled, a user's identity can be assumed by passing it in on the query string"""
+    app.logger.debug("Getting user")
+
+    user = None
+
+    if app.config.get("LOGIN_DISABLED") or current_user.is_admin:
+        user_id = request.args.get("user")
+        if user_id:
+            user = User.query.filter(User.user_id == user_id).first()
+            if not user:
+                abort(400, description="Provided user does not exist!")
+
+    if not user:
+        user = current_user
+
```

Comment:
user lacks the `is_admin` property when we login anonymously (via setting LOGIN_DISABLED = True in the config), resulting in endpoints 500ing. From my testing this can be fixed by setting the property to be true if it doesn't exist

```
curl localhost:5000/api/families/1
{
  "error": "'AnonymousUserMixin' object has no attribute 'is_admin'"
}
```